Client/Frontend/Widgets/ThumbnailCell.swift

```diff
@@ -323,4 +246,13 @@ class ThumbnailCell: UICollectionViewCell {
             }
         }
     }
+
+    func extractImageFromCell() -> UIImage? {
```

Comment:
This looks like it's only for the TopSites stuff, I think there's already an image property stored somewhere in the cell... I'm not sure but I would ask @farhanpatel since he's been working more closely with it recently. But if there's no way to just get the image/it isn't stored anywhere per cell, then this is fine.